### PR TITLE
Responsive voting app

### PR DESCRIPTION
**Pull request by AquiGorka**

### CODE REVIEW

File: apps/voting/app/src/App.js [hunk 1]
```diff
@@ -244,9 +245,16 @@ class App extends React.Component {
                   </Title>
                 }
                 endContent={
-                  <Button mode="strong" onClick={this.handleCreateVoteOpen}>
-                    New Vote
-                  </Button>
+                  <React.Fragment>
+                    <BreakPoint to="medium">
+                      <NewVoteButton onClick={this.handleCreateVoteOpen} />
```
Comment (bpierre) [inline apps/voting/app/src/App.js]: We should add a title:

```suggestion
                      <NewVoteButton
                        title="New Vote"
                        onClick={this.handleCreateVoteOpen}
                      />
```

Comment (bpierre) [inline apps/voting/app/src/App.js]: Same remark [than on the Token Manager](https://github.com/aragon/aragon-apps/pull/612/files#r248775882) about this button name: shall we change it, or have an icon mode + a label mode in the button itself?

**Comment by AquiGorka** [comment]
Thank you @bpierre 😄 all comments addressed.

File: apps/voting/app/src/components/VotePanelContent.js [hunk 2]
```diff
@@ -340,6 +327,17 @@ class VotePanelContent extends React.Component {
   }
 }
 
+const ResponsiveVotePanelContent = props => (
+  <React.Fragment>
+    <BreakPoint to="medium">
+      <VotePanelContent {...props} shorten />
```
Comment (sohkai) [inline apps/voting/app/src/components/VotePanelContent.js]: Similar to https://github.com/aragon/aragon/pull/548/files#r249327353, but what do you think about making this more responsive as `shortAddresses`?

**Comment by sohkai** [comment]
@AquiGorka Also removed the title's right margin in 18a4d21 (although we can introduce it later when we figure out the title badges: https://github.com/aragon/aragon-apps/issues/633).

**Comment by AquiGorka** [comment]
@bpierre added the `min-width` suggestion.

**Comment by AquiGorka** [comment]
@jounih the only thing stopping me from merging this PR is what you found on the margins between the cards but I'm not able to reproduce.

**Pull request merged**